r1cs/gadgets/std/src/groups/group_vec.rs

```diff
@@ -277,6 +277,29 @@ impl<ConstraintF: Field, G: Group, GG: GroupGadget<G, ConstraintF>> GroupGadget<
         Ok(Self::new(res_vec))
     }
 
+    // pointless to implement it, would be equivalent to GG::mul_bits_fixed_base
+    fn mul_bits_fixed_base<CS: ConstraintSystemAbstract<ConstraintF>>(_base: &G, _cs: CS, _bits: &[Boolean]) -> Result<Self, SynthesisError> {
```

Comment:
If these are supposed to call the default implementation then you must not overwrite them and put unimplemented!(); otherwise I don't understand what you mean here